### ISSUE TITLE
SetuptoolsDeprecationWarning: Invalid dash-separated options

### ISSUE THREAD

**Issue by orlitzky** [james-d-mitchell/gaplint#29]
This is trivial, but newer setuptools complain...

```
/usr/lib/python3.11/site-packages/setuptools/dist.py:476: SetuptoolsDeprecationWarning: Invalid dash-separated options
!!

        ********************************************************************************
        Usage of dash-separated 'description-file' will not be supported in future
        versions. Please use the underscore name 'description_file' instead.

        By 2024-Sep-26, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.

        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
```

The offending line is,

https://github.com/james-d-mitchell/gaplint/blob/main/setup.cfg#L4

**Comment by james-d-mitchell** [comment]
Thanks for the report @orlitzky this one is easy to fix sounds like!

**Comment by james-d-mitchell** [comment]
Resolved in v1.2.0
